apps/docs-website/docs/core-philosophy/criteria.md

```diff
@@ -5,16 +5,16 @@ description: 🎯 What line of reasoning drives Sheriff development
 
 # Criteria
 
-This library is very opinionated, but it's for the better. A lot of decisions were taken so you don't have to.<br />
-You can now quickstart static analysis in all your Typescript projects with ease. <br />
+This library is very opinionated, but for the better. A lot of decisions were taken so you don't have to.<br />
+You can now quickstart static analysis in all your TypeScript projects with ease.<br />
 Just type `npm init @sherifforg/config` in a CLI and you are good to go.
 
-Easiness of use without compromises is the top priority of Sheriff. The basic principle behind many design decisions of Sheriff is to require as less inputs from the user as possible. <br />
-You can think of Sheriff like Prettier or `create-react-app`. It's a tool that comes battery-packed with optimal defaults. It removes configuration decisions from the equation, so you or your team can focus on developing the actual product.<br />
-And if you don't like something, you can easily override it, and just as easily you can extend it. See: [configuration](../configuration.mdx).
+Ease of use without compromises is the top priority of Sheriff. The basic principle behind many design decisions of Sheriff is to require as less inputs from the user as possible.<br />
+You can think of Sheriff like Prettier. It’s a tool that comes battery-packed with optimal defaults. It removes configuration decisions from the equation, so you or your team can focus on developing the actual product.<br />
+If you disagree with something, you can easily override it, and you can extend it just as easily. See: [configuration](../configuration.mdx).
 
 This config is particularly useful for big teams with developers of various skill levels.<br />
-Sheriff was made to prevent all kind of mistakes and to align the team on the same playing field. It is battle-tested in real-world scenarios and shines especially in such.<br />
-Also, Sheriff target pretty much everybody. Junior developers can learn a lot by using it, and Lead developers can save a lot of time on configuration and maintenance.
+Sheriff was made to prevent all kind of mistakes and to align teams stylistically. It is battle-tested in real-world scenarios and shines especially in such.<br />
```

Comment:
👎

Reply:
What if I omit `stylistically`? "aligning on the playing field" isn't really an applicable metaphor here.

Reply:
I came up with this:

```suggestion
Sheriff was made to prevent all kinds of mistakes by aligning teams. It is battle-tested in real-world scenarios and shines especially in such.<br />
```

Reply:
> "aligning on the playing field" isn't really an applicable metaphor here.

Why not? I like it

Reply:
ChatGPT agrees with you, so I'll quit pushing this one. It alludes to the metaphor of "evening the playing field", but apparently "aligning the playing field" is slightly different.

Reply:
i think the only correct fix here is kind > kinds